kso_utils/tutorials_utils.py

```diff
@@ -433,7 +433,7 @@ def extract_clips(
         full_prompt = init_prompt
         mod_prompt = ""
         output_prompt = ""
-        def_output_prompt = f".output('{str(output_clip_path)}', ss={str(upl_second_i)}, t={str(clip_length)}, movflags='+faststart', crf=20, pix_fmt='yuv420p', vcodec='libx264')"
```

Comment:
Why did you remove the vcodec? 
I did not look really carefully at the error messages of the tests from the previous commit, but I did not see anything about tutorial_utils I think.
And I thought that the vcodec is needed for ffmpeg to know what codec to use. Then it should not work without, or use a default that we maybe do not want to use.

Reply:
I see that you added it later again, maybe squash the arguments so that it is not visible in the history.